core/example/scafacos_example.cpp

```diff
@@ -0,0 +1,444 @@
+#include <Cabana_Types.hpp>
+#include <Cabana_AoSoA.hpp>
+#include <Cabana_MemberTypes.hpp>
+
+#include <Kokkos_Core.hpp>
+
+#include <cstdlib>
+#include <iostream>
+#include <string>
+
+#include "fcs.h"
```

Comment:
I don't remember any coding guidelines for Cabana but this is a bit inconsistent. I would expect you to use angle brackets and place it above the includes from the standard library.